webrtc.html

```diff
@@ -1367,6 +1368,142 @@
+              the new <a>ICE transports setting</a> changes the existing
+              setting, no action will be taken until the next gathering
+              phase. If a script wants this to happen immediately, it
+              should do an ICE restart.</p>
+            </li>
+            <li>
+              <p>Set the <a>ICE Agent</a>'s prefetched <dfn>ICE candidate
+              pool size</dfn> as defined in <span data-jsep=
+              "ice-candidate-pool constructor">[[!JSEP]]</span> to the
+              value of <code><var>configuration</var>.<a data-for=
+              "RTCConfiguration">iceCandidatePoolSize</a></code>. If the
+              new <a>ICE candidate pool size</a> changes the existing
+              setting, this may result in immediate gathering of new
+              pooled candidates, or discarding of existing pooled
+              candidates, as defined in <span data-jsep=
+              "setconfiguration">[[!JSEP]]</span>.</p>
+            </li>
+            <li>
+              <p>Let <var>validatedServers</var> be an empty list.</p>
+            </li>
+            <li>
+              <p>If <code><var>configuration</var>.<a data-for=
+              "RTCConfiguration">iceServers</a></code> is defined, then
+              run the following steps for each element:</p>
+              <ol>
+                <li>
+                  <p>Let <var>server</var> be the current list
+                  element.</p>
+                </li>
+                <li>
+                  <p>If <code><var>server</var>.urls</code> is a string,
+                  let <code><var>server</var>.urls</code> be a list
+                  consisting of just that string.</p>
+                </li>
+                <li>
+                  <p>For each <var>url</var> in
+                  <code><var>server</var>.urls</code> parse
+                  <var>url</var> and obtain <var>scheme name</var>. If
+                  the <var>scheme name</var> is not implemented by the
+                  browser, or if parsing based on the syntax defined in
+                  [[!RFC7064]] and [[!RFC7065]] fails, <a>throw</a> a
+                  <code>SyntaxError</code>.</p>
+                </li>
+                <li>
+                  <p>If <var>scheme name</var> is <code>turn</code> or
+                  <code>turns</code>, and either of
+                  <code><var>server</var>.username</code> or
+                  <code><var>server</var>.credential</code> are omitted,
+                  then <a>throw</a> an <code>InvalidAccessError</code>.</p>
+                </li>
+                <li>
+                  <p>If <var>scheme name</var> is <code>turn</code> or
+                  <code>turns</code>, and
+                  <code><var>server</var>.credentialType</code> is
+                  <code>"password"</code>, and
+                  <code><var>server</var>.credential</code> is not a
+                  <span class="idlMemberType"><a>DOMString</a></span>, then
```

Comment:
This is the only change in this section apart from moving, respec was complaining about DOMString. Not sure if idlMemberType is right but it made respec happy.